cmake/FindPostgreSQL.cmake

```diff
@@ -51,7 +51,11 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
-include(${CMAKE_SOURCE_DIR}/cmake/OpenSSL.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/OpenSSL.cmake RESULT_VARIABLE file_found)
+
+if(file_found STREQUAL "NOTFOUND")
+    message(FATAL_ERROR "cant find openssl.cmake file. It is expected to be under cmake directory")
```

Comment:
Do we need this if `include` itself can fail?

Reply:
Does this particular change contribute to fixing the issue? If not, I suggest leaving it out and handling failed includes (any) more generically.

Reply:
Yes, this change does fix the current issue.

Reply:
I didn't mean the `include` itself but the error reporting. Given that we do so many imports everywhere, handling it just here this way doesn't seem to be right.

Reply:
I assume `include` does "send_error" which doesn't make cmake fail right away, but prints the error and exits with an error.

You're just handling the case by converting that "send_error" to "fatal_error"

I suggest we extract the discussion whether file inclusion should be a fatal error or not into a separate issue and come up with a universal solution instead of an ad-hoc handler.

Reply:
Alright. Updated to remove the error handling.